src/main.c

```diff
@@ -982,6 +983,22 @@ int main_client (int argc, char **argv, GromitData *data)
            else
              data->clientdata = "-1"; /* default to grab all */
          }
+       else if (strcmp (arg, "-l") == 0 ||
+           strcmp (arg, "--line") == 0)
+         {
+           if (argc - (i+1) == 6) /* this command must have exactly 6 params */
```

Comment:
This could use some more failsafe testing, i.e. that all args are of the correct type.